config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -598,6 +598,25 @@ postsubmits:
         resources:
           requests:
             memory: "1Gi"
+  - name: post-test-infra-upload-triage
+    cluster: test-infra-trusted
+    branches:
+    - master
+    run_if_changed: '^triage/Makefile$|^triage/[^/]+(\.html|\.js|.css)$'
+    decorate: true
+    spec:
+      serviceAccountName: pusher
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-bazel:v20200520-v0.4-30-g9d6313a
+        command:
+        - make
+        args:
+        - push-static
+        - push-staging
+        - push-go
+        resources:
+          requests:
+            memory: "1Gi"
     annotations:
```

Comment:
I think you stole this from `post-test-infra-upload-oncall` by mistake, looks like they both need an `annotations`

Reply:
heh I just pushed that fix

Reply:
`post-test-infra-upload-oncall` still doesn't have its `annotations` though :)